README.md

```diff
@@ -61,6 +52,19 @@ Windows may complain about security due to code not being signed, this will be c
 
 [More tips](https://flow-launcher.github.io/docs/#/usage-tips)
 
+### Integrations
+  - If you use Python plugins:
+    - Install [Python3](https://www.python.org/downloads/), download `.exe` installer.
+    - Add Python to `%PATH%` or set it in flow's settings.
+    - Use `pip` to install `flowlauncher`, open cmd and type `pip install flowlauncher`.
+    - The Python plugin may require additional modules to be installed, please ensure you check by visiting the plugin's website via `pm install` + plugin name, go to context menu and select `Open website`. 
+    - Start to launch your Python plugins.
+  - Flow searches files and contents via Windows Index Search, to use Everything: `pm install everything`.
+
+## Plugins
+
+There is a [list of actively maintained plugins](https://github.com/Flow-Launcher/docs/Plugins.md) in the documentation section, some of which are integrated from other launchers.
+
```

Comment:
instead of a separate section, could this be just a bullet point in Integrations section please

Reply:
it can though i made it a separate thing because I'm not sure an average user would think to look for plugin installation information under an "integrations" section and the extensibility is a core Flow feature. (my focus generally is documentation UX). But happy to do whatever.

Reply:
Good point. Let's keep this section then, once the python auto install feature is rolled out I might merge Integration into the Plugin section as well so we can just use it as a quick guide on using plugins and where to find more info